package.json

```diff
@@ -22,6 +24,7 @@
     "test": "watai test && watai test/bootstrap-compat",
     "precompile": "rimraf dist/*",
     "compile": "webpack --optimize-minimize",
-    "compile-continuously": "webpack --watch"
+    "compile-continuously": "webpack --watch",
```

Comment:
webpack-dev-server automatically reloads the page, and doesn't write to dist/, so it's a different approach, that will probably be used instead of --watch, yes